### PR TITLE
Add hooks for rendering before and after the user menu

### PR DESCRIPTION
**Pull request by nbejansen**
Added hooks for rendering views before and after the user menu. Use case: team switcher / quick-add.

### CODE REVIEW

File: packages/admin/docs/08-appearance.md [hunk 1]
```diff
@@ -313,3 +313,5 @@ The available hooks are as follows:
 - `styles.end` - after styles are defined
 - `global-search.start` - before [global search](resources/global-search) input
 - `global-search.end` - after [global search](resources/global-search) input
+- `user-menu.start` - before [user menu](navigation) input
```
Comment (danharrin) [inline packages/admin/docs/08-appearance.md:316]:
```suggestion
- `user-menu.start` - before [user menu](navigation#customizing-the-user-menu) input
```

**Comment by danharrin** [comment]
You've blocked maintainers from being able to edit the PR somehow, so you'll need to fix that docs link yourself before I can merge :)

**Comment by nick-jansen** [comment]
Thank you for your review @danharrin, i've updated the docs.

**Comment by danharrin** [comment]
Thanks

**Pull request merged**